@types/ucontent.d.ts

```diff
@@ -0,0 +1,23 @@
+declare module "ucontent" {
+  type TemplateFunction<T> = (
+    template: TemplateStringsArray,
+    ...values: unknown[]
+  ) => T;
+
+  class UContent extends String {
```

Comment:
Q: why "UContent"? It's not immediately obvious what that means. Maybe just needs a comment.

Reply:
Umm.. because we're defining types for `uncontent` package?

Reply:
Ok, but what *is* `UContent`... in particular, what's the "U" standard for?

Reply:
U is short for micro (mu). It's the package name. Am I missing something?

Reply:
Oh, I see now... yeah, somehow I missed the whole "mu" package inclusion. Seems I forgot we were using that. Just seems odd that we would be defining types of external packages, which is also why I got confused.

Reply:
Don't need this file with https://github.com/marcoscaceres/respec.org/pull/176/commits/7947b2e949ff60d2a9d64edcc23f116956143dda :tada:

Reply:
Unless comment turned out to be useful! :D